crm_phonecall/models/res_config_settings.py

```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    group_show_form_view = fields.Boolean(
+        "Enable form view for phone calls",
+        help="By default form is disabled for calls, with this group it is enabled.",
+        implied_group="crm_phonecall.group_show_form_view",
+        default=True,
+    )
```

Comment:
You need to add this to the res.config view as well

You can add a new section "Phonecalls" there
![image](https://user-images.githubusercontent.com/1914185/80995649-dfad1400-8e3e-11ea-82fc-add10b94bc82.png)